packages/frontend-api/src/Model/Resolver/Order/PromoCode/PromoCodeResolverMap.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrontendApiBundle\Model\Resolver\Order\PromoCode;
+
+use Overblog\GraphQLBundle\Resolver\ResolverMap;
+use Shopsys\FrontendApiBundle\Model\Order\PromoCode\PromoCodeWithDiscount;
+
+class PromoCodeResolverMap extends ResolverMap
+{
+    /**
+     * @return array
+     */
+    protected function map(): array
+    {
+        return [
+            'PromoCode' => [
```

Comment:
I believe, for better clarity, it would be better to name the API type "PromoCodeWithDiscount(+Decorator)" as well

Reply:
I actually do not think so. Objects in API are not same objects as in backend. API users do not need to know, that our `PromoCode` entity does not include discount. They need `PromoCode` as final product :D  For example we call `CartWithModificationsResult` just a `Cart`.

Reply:
ok, I can live with that